pkg/network/protocols/http/testutil/pythonserver.go

```diff
@@ -86,7 +86,11 @@ func writeTempFile(pattern string, content string) (*os.File, error) {
 	if err != nil {
 		return nil, err
 	}
-	defer f.Close()
+	defer func() {
+		if err := f.Close(); err != nil {
+			log.Printf("Error closing file: %s\n", err)
```

Comment:
please use the datadog logger instead

Reply:
nice catch, it should be done along with the other suggestions.